src/AppInstallerCLICore/ExecutionArgs.h

```diff
@@ -218,10 +218,29 @@ namespace AppInstaller::CLI::Execution
             return types;
         }
 
+        // If the user passes the same value multiple times inside a MultiQuery, operations will be repeated
+        // Since there currently is not a way to include search options within a MultiQuery, processing duplicates
+        // does not make sense within a single invocation
+        void MakeMultiQueryContainUniqueValues()
+        {
+            auto itr = m_parsedArgs.find(Type::MultiQuery);
+            
+            // If there is not a value in MultiQuery, or there is only one value, it is presumed to be unique
+            if (itr == m_parsedArgs.end() || itr->second.size() == 1)
+            {
+                return;
+            }
+
+            std::set<std::string> querySet;
+            std::vector<std::string> queryStrings = itr->second;

```

Comment:
```suggestion
            std::vector<std::string>& queryStrings = itr->second;

```

And remove line 238 to avoid unnecessary copies